language/google/cloud/language/api_responses.py

```diff
@@ -0,0 +1,116 @@
+from google.cloud.language.syntax import Token
+
+
+class EntityResponse(object):
+    """A representation of a response sent back from the
+    ``analyzeEntites`` request to the Google Natural language API.
+
+    :type entities: list
+    :param entities: A list of :class:`~.language.entity.Entity` objects.
+
+    :type language: str
+    :param language: The language used for analysis.
+    """
+    def __init__(self, entities, language):
+        self.entities = entities
+        self.language = language
+
+    @classmethod
+    def from_api_repr(cls, payload):
+        """Return an entity response from a JSON representation.
+
+        :type payload: dict
+        :param payload: A dictionary representing the response.
+        """
+        return cls(
+            entities=[Entity.from_api_repr(i) for i in payload['entities']],
+            language=payload['language'],
+        )
+
+
+class SentimentResponse(object):
+    """A representation of a response to an ``analyzeSentiment`` request
+    to the Google Natural Language API.
+
+    :type sentiment: :class:`~.language.sentiment.Sentiment`
+    :param sentiment: A Sentiment object.
+
+    :type language: str
+    :param: language: The language used for analyzing sentiment.
+
+    :type sentences: list
+    :param sentences: A list of :class:`~.language.syntax.Sentence` objects.
+    """
+    def __init__(self, sentiment, language, sentences):
+        self.sentiment = sentiment
+        self.language = language
+        self.sentences = sentences
+
+    @classmethod
+    def from_api_repr(cls, payload):
+        """Return an sentiment response from a JSON representation.
+
+        :type payload: dict
+        :param payload: A dictionary representing the response.
+        """
+        return cls(
+            language=payload.get('language', None),
+            sentences=[Sentence.from_api_repr(i) for i
+                       in payload.get('sentences', ())],
+            sentiment=Sentiment.from_api_repr(payload['documentSentiment']),
+        )
+
+
+class SyntaxResponse(object):
+    """A representation of a response to an ``analyzeSyntax`` request
+    to the Google Natural Language API.
+
+    :type tokens: list
+    :param tokens: A list of :class:`~.language.syntax.Token` objects.
+
+    :type language: str
+    :param: language: The language used for analyzing sentiment.
+
+    :type sentences: list
+    :param sentences: A list of :class:`~.language.syntax.Sentence` objects.
+    """
+    def __init__(self, tokens, language, sentences):
+        self.tokens = tokens
+        self.language = language
+        self.sentences = sentences
+
+    @classmethod
+    def from_api_repr(cls, payload):
+        """Return an syntax response from a JSON representation.
+
+        :type payload: dict
+        :param payload: A dictionary representing the response.
+        """
+        return cls(
+            language=payload.get('language', None),
```

Comment:
Oh hey! Today I learned. I thought that `{}.get('foo')` was equivalent to `{}['foo']`, not `{}.get('foo', None)`. I _think_ I prefer the more explicit format though (but I might be biased by not knowing what the shorter form did).